ethicml/data/tabular_data/admissions.py

```diff
@@ -0,0 +1,116 @@
+"""Class to describe the 'UFRGS Entrance Exam and GPA Data'.
+
+Persistent link: https://dataverse.harvard.edu/dataset.xhtml?persistentId=doi:10.7910/DVN/O35FW8
+
+Data Description: Entrance exam scores of students applying to a university in Brazil
+(Federal University of Rio Grande do Sul), along with the students' GPAs during the first three
+semesters at university. In this dataset, each row contains anonymized information about an
+applicant's scores on nine exams taken as part of the application process to the university, as
+well as their corresponding GPA during the first three semesters at university.
+The dataset has 43,303 rows, each corresponding to one student.
+The columns correspond to:
+1) Gender. 0 denotes female and 1 denotes male.
+2) Score on physics exam
+3) Score on biology exam
+4) Score on history exam
+5) Score on second language exam
+6) Score on geography exam
+7) Score on literature exam
+8) Score on Portuguese essay exam
+9) Score on math exam
+10) Score on chemistry exam
+11) Mean GPA during first three semesters at university, on a 4.0 scale.
+
+We replace the mean GPA with a binary label Y representing whether the student’s GPA was above 3.0
+
+
+```bibtex
+@data{DVN/O35FW8_2019,
+  author    = {Castro~da~Silva, Bruno},
+  publisher = {Harvard Dataverse},
+  title     = {{UFRGS Entrance Exam and GPA Data}},
+  UNF       = {UNF:6:MQqEQGXiIfQTbS7q9QJ5uw==},
+  year      = {2019},
+  version   = {V1},
+  doi       = {10.7910/DVN/O35FW8},
+  url       = {https://doi.org/10.7910/DVN/O35FW8}
+}
+```
+"""
+
+from typing_extensions import Literal
+
+from ..dataset import Dataset
+from ..util import deprecated
+
+__all__ = ["Admissions", "admissions"]
+
+AdmissionsSplits = Literal["Gender"]
+
+
+@deprecated
```

Comment:
@olliethomas why do you keep adding deprecated code? 😄 deprecated code is there to be removed! not added!

Reply:
or are you trying to send the hidden message that you prefer the capitalized version of the function?

Reply:
only because I don't know what's happening with the uppercase vs lowercase.

Reply:
> or are you trying to send the hidden message that you prefer the capitalized version of the function?
I think it's safe to assume that there is no hidden message and that I'm mostly just copying the adult dataset file ever time I add a new dataset